src/wp-includes/functions.php

```diff
@@ -175,10 +175,12 @@ function wp_maybe_decline_date( $date ) {
 	 * translate this to 'on'. Do not translate into your own language.
 	 */
 	if ( 'on' === _x( 'off', 'decline months names: on or off' ) ) {
+
+		$months          = $wp_locale->month;
+		$months_genitive = $wp_locale->month_genitive;
+
```

Comment:
This moving of two lines is not part of the 45611 changeset. I think it's from the big date changes done in WP 5.3.

Reply:
This is part of 45555 also.